### PR TITLE
added YouTube2.0 branch and code

### PR DESCRIPTION
**Pull request by SoftDevRaj**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,11 +19,10 @@
     "googleapis": "^126.0.1",
     "method-override": "^3.0.0",
     "mysql2": "3.6.1",
+    "node-fetch": "^3.3.2",
     "passport": "^0.6.0",
     "passport-local": "^1.0.0",
     "request": "2.88.2",
-    "dotenv": "16.3.1",
```
Comment (DupeaJ) [inline package.json:25]: we need dependancy for dotenv

**Review by DupeaJ — COMMENTED** [review]
great seperation of code, we need to include dotenv in dependancys though.

**Review by DupeaJ — COMMENTED** [review]
adds youtube api

**Review by sophiawilliams16 — APPROVED** [review]
Great work!!

**Pull request merged**